src/connectors/index.ts

```diff
@@ -38,20 +38,10 @@ export const injected = new InjectedConnector({
 
 const rpcForWalletConnect: {
   [chainId: number]: string
-} =
-  CHAIN_ID === 4
-    ? { 4: NETWORK_URL_RINKEBY }
-    : CHAIN_ID === 100
-    ? { 100: NETWORK_URL_XDAI }
-    : { 1: NETWORK_URL_MAINNET }
+} = { 4: NETWORK_URL_RINKEBY, 100: NETWORK_URL_XDAI, 1: NETWORK_URL_MAINNET }
 
-// The WalletConnectConnector only supports a single RPC when configured,
-// so we are going to use the RPC associated with the Chain_Id environment variable configuration
-// Doing this, we will support every network, including xDai
 export const walletconnect = new WalletConnectConnector({
-  rpc: {
-    ...rpcForWalletConnect,
-  },
+  rpc: rpcForWalletConnect,
```

Comment:
I think handing over many id's is risky, as some wallets will then deny the connection, if not all ids are supported